frontend/src/services/UserService.ts

```diff
@@ -19,26 +19,44 @@ interface UserDataWithToken extends IUserData {
   token: string;
 }
 
-interface LoginUserData {
-  errors?: any;
+interface LoginUserPayload {
+  error?: any;
   user?: UserDataWithToken;
 }
 
 interface LoginUserResponse {
   status: number;
-  data: LoginUserData;
+  data: LoginUserPayload;
 }
 
 export const loginUser = (user: ILoginData): Promise<LoginUserResponse> =>
-  fetch("/api/users/login", {
+  fetch("/api/users/sign_in", {
     method: "POST",
     body: JSON.stringify({ user: user }),
     headers: {
       "Content-Type": "application/json",
     },
+  }).then(async response => {
+    const data: LoginUserPayload = await response.json();
+    const isInvalidCreds =
+      data.error && data.error["email or password"] !== undefined;
+    const status = isInvalidCreds ? 401 : response.status;
+
+    console.log("USER: ", data);
+
+    return {
+      status,
+      data,
+    };
+  });
+
+export const logoutUser = () => {
+  fetch("/api/users/sign_out", {
+    method: "DELETE",
   }).then(response => {
-    return { status: response.status, data: response.json() as LoginUserData };
```

Comment:
no data is returned when logging out, hence doing response.json() throws an error